exercises/arrays/allPositive/allPositive.js

```diff
@@ -12,14 +12,37 @@
  *   and false otherwise.
  */
 function allPositive(array) {
+  for (let index = 0; index < array.length; index++) {
+    if (array[index] <= 0) {
+      return false;
+    }
+  }
+  return true;
```

Comment:
Great use of the for loop and return statements.  You may want to double check your if statement.  Is zero considered to be negative?

Reply:
Thank you! I believe in the example, 0 was considered to be negative or not positive

Reply:
`0` is neither positive nor negative.

So "Are any negative?" is a different question than "Are all positive?"

Reply:
Right, so the output would still be false right?

Reply:
Yep, `allPositive([0, 0, 1])` should return `false` (and yours does).

Reply:
Agreed, sorry for the confusion, my bad math :)